src/core/migrator.ts

```diff
@@ -70,6 +71,32 @@ export class Migrator extends Umzug implements Generator {
     this.attributeName = attributeName;
   }
 
+  private async _handleTableCreation(client: any, params: any) {
+    const tablesResult = await client.listTables().promise();
+    const tables = tablesResult.TableNames;
+    if (tables.length > 0 && !tables.includes(this.tableName)) {
+      await client.createTable(params).promise();
+    }
+  }
+
+  private async _awaitTableActiveStatus(client: any, retries = 5) {
```

Comment:
this feels a bit close to me, maybe give it more time than 5 seconds by default?

Reply:
sure thing, i tested with 10 initially but found it was always active after 5 seconds. but will up the threshold